vrs/Dockerfile

```diff
@@ -0,0 +1,105 @@
+# Dockerfile for creating an image that can run the VRS Annotation Script inside of it
+# includes libpqdev and pythondev, plus psycopg2, ga4gh.vrs[extras] , and seqrepo
+# TODO: incorporate 'click' and custom Validation Error from ga4gh.vrs.extras.translator
+
+FROM 'google/cloud-sdk:slim'
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    apt-utils \
+    docker-compose \
+    g++ \
+    gawk \
+    less \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    liblzma-dev \
+    libncurses5-dev \
+    man-db \
+    pkg-config \
+    python3-venv \
+    software-properties-common \
+    unzip \
+    wget \
+    zlib1g-dev \
+    libpq-dev \ 
+    python-dev\
+    && \
+	# clean up apt cache
+    rm -rf /var/lib/apt/lists/*
+
+#install Java 8 for hail
+RUN wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | apt-key add - \ 
+    && add-apt-repository -y https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/ \ 
+    && apt-get update && apt-get install -y adoptopenjdk-8-hotspot
+
+RUN mkdir tools 
+WORKDIR /tools
+
+
+
+# install vcftools
+ENV VCFTOOLS_VERSION="0.1.16"
+RUN wget https://github.com/vcftools/vcftools/releases/download/v${VCFTOOLS_VERSION}/vcftools-${VCFTOOLS_VERSION}.tar.gz \
+    && tar xf vcftools-${VCFTOOLS_VERSION}.tar.gz \
+    && rm vcftools-${VCFTOOLS_VERSION}.tar.gz \
+	&& cd vcftools-${VCFTOOLS_VERSION} \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& make clean
+
+
+# Install python packages
+RUN apt-get update \
+	&& apt-get dist-upgrade -y \
+	&& apt-get install -y --no-install-recommends\
+    libc6-dev \
+    libffi-dev \
+    libgdbm-dev \
+    liblapack-dev \
+    liblapack3 \ 
+    libncursesw5-dev \
+    libopenblas-base \
+    libopenblas-dev \
+    libsqlite3-dev \
+    libssl-dev \
+    openssl \
+    python3-smbus \
+    python3 python3-pip
+
+# Upgrade pip to latest version
+RUN python3 -m pip install --upgrade pip
+
+# Install hail and other python libraries
+# including packages needed for VRS annotation functionality
+ENV HAIL_VERSION="0.2.110"
```

Comment:
we're eventually going to want to change this version

https://atgu.slack.com/archives/C033C0MMM/p1678981995419869

https://the-tgg.slack.com/archives/CAAQQ30CX/p1679080838951189